app/views/kaminari/_gap.html.slim

```diff
@@ -0,0 +1,9 @@
+/ Non-link tag that stands for skipped pages...
+  - available local variables
+    current_page : a page object for the currently displayed page
+    total_pages  : total number of pages
+    per_page     : number of items to fetch per page
+    remote       : data-remote
+span.page.gap
+  == t('views.pagination.truncate').html_safe
```

Comment:
may be you can try to extract html to view and leave only text for locales? It's not good to keep html in locales